.github/workflows/review.yml

```diff
@@ -23,13 +23,11 @@ jobs:
           if git diff --name-only origin/${{ github.base_ref }} HEAD | grep -Eq '*\.js|node\.d\.plugin\.in' ; then
             echo 'run_eslint=1' >> $GITHUB_ENV
           fi
-      - name: Run eslint
+      - name: Install eslint
+        run: npm install eslint -D
```

Comment:
This needs the same `if: env.run_eslint == 1` conditional as the next step has, otherwise we’ll always install eslint instead of only doing so when we actually need to.